services/baseremotecontrol/builtin/builtin_test.go

```diff
@@ -32,6 +32,14 @@ func TestBaseRemoteControl(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	testutils.VerifySameElements(t, depNames, []string{"baseTest", "inputTest"})
 
+	cfg.ControlModeName = "fail"
+	_, err = cfg.Validate("")
+	test.That(t, err, test.ShouldBeError,
+		errors.Errorf(
+			`Error validating. Path: "" Error: Control mode '%s' is not in %v`,
+			cfg.ControlModeName,
+			modes))
```

Comment:
I chose to use `Errorf` here instead of `resource.NewConfigValidationError` to be consistent with the other error-related test cases.

Reply:
Yeah, I'm trying to kill useages. Errorf because it's a deprecated public archive package, so consistency isn't great here. Using the error itself is more idiomatic.

Reply:
If you want to.